cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt

```diff
@@ -113,6 +113,8 @@ credentials "auth_aws" do
   aws_account_number $param_aws_account_number
 end
 
+auth "auth_rs", type: "rightscale"
```

Comment:
shouldn't this be something like this instead?
`credentials "auth_rs" do
  schemes "oauth2"
  label "Optima_API"
  description "Select FlexeraOne OAuth2 credentials"
  tags "provider=flexera"
end`

To account for both NAM and EU shards?